mcs/class/corlib/System.Reflection/MonoMethod.cs

```diff
@@ -572,6 +572,12 @@ internal void SerializationInvoke (Object target, SerializationInfo info, Stream
 			Invoke (target, new object[] { info, context });
 		}
        #endregion
+
+		// This is required to support certain kinds of serialization
```

Comment:
Is there another comment that would be more accurate (as to why all RuntimeConstructorInfo's are SecurityCritical?)